provider/repodata/data/json/locid_transform/script_dir@1/und.json

```diff
@@ -0,0 +1,170 @@
+{
+  "rtl": {
+    "Adlm": "RightToLeft",
+    "Aghb": "LeftToRight",
```

Comment:
Discussion/Thought/Suggestion: We could save a bunch of space if we only stored RightToLeft and not LeftToRight, but then we'd lose the ability to return `None` if we don't have data for that script. Is that an important feature?

Reply:
We have 166 entries, so even with the map metadata we would save <200B. I don't think that's a good tradeoff.

We could store one LTR and one RTL list, but then we would have two binary searches (although `is_ltr` and `is_rtl` would only need one each).

Reply:
I think returning `None` is an important functionality, otherwise we'd have to return either RTL or LTR for `Zzzz`.

Reply:
I switched to the proposed list approach, saving 166B. 

For the general get-direction method it raises the question if the LTR or RTL list should be searched first. I went with LTR because there's more LTR scripts, maybe meaning we'll have more hits in LTR. OTOH, clients of `LocaleDirectionality` might be biased to work with RTL scripts more often, so I'm not sure.  

Also, there are 35 RTL and 131 LTR scripts.